src/checkbox/__tests__/checkbox.test.tsx

```diff
@@ -34,6 +34,12 @@ test('renders an input element', () => {
   expect(nativeInput.type).toBe('checkbox');
 });
 
+test('can be marked as required', () => {
```

Comment:
should we add the same test for the toggle component?

Reply:
good catch - I'd got my revisions mixed up, and this should actually only apply to the Checkbox component, updated